src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/IUnitTestingIncrementalAnalyzerProvider.cs

```diff
@@ -6,6 +6,10 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.SolutionCrawler
 {
     internal interface IUnitTestingIncrementalAnalyzerProvider
     {
-        IUnitTestingIncrementalAnalyzer? CreateIncrementalAnalyzer(Workspace workspace);
+        IUnitTestingIncrementalAnalyzer? CreateIncrementalAnalyzer(
+#if false // Not used in unit testing crawling
+            Workspace workspace
+#endif
+            );
```

Comment:
Have validated that unittesting's incremental analyzer does not use this: http://index/?rightProject=Microsoft.CodeAnalysis.Features&file=ExternalAccess%5cUnitTesting%5cAPI%5cUnitTestingIncrementalAnalyzerProvider.cs&line=26